docs/operations/api-reference.md

```diff
@@ -293,10 +293,10 @@ Returns full segment metadata for a specific segment in the cluster.
 
 Return the tiers that a datasource exists in.
 
-#### Note for coordinator's POST and DELETE API's
-The segments would be enabled when these API's are called, but then can be disabled again by the coordinator if any dropRule matches. Segments enabled by these API's might not be loaded by historical processes if no loadRule matches.  If an indexing or kill task runs at the same time as these API's are invoked, the behavior is undefined. Some segments might be killed and others might be enabled. It's also possible that all segments might be disabled but at the same time, the indexing task is able to read data from those segments and succeed.
+#### Note for coordinator's POST and DELETE APIs
+The segments would be enabled when these APIs are called, but then can be disabled again by the coordinator if any dropRule matches. Segments enabled by these APIs might not be loaded by historical processes if no loadRule matches.  If an indexing or kill task runs at the same time as these APIs are invoked, the behavior is undefined. Some segments might be killed and others might be enabled. It's also possible that all segments might be disabled but at the same time, the indexing task is able to read data from those segments and succeed.
```

Comment:
Wonder if we can update this paragraph while we're at it. If my understanding is correct, perhaps something like the following:

```suggestion
While segments may be enabled by issuing POST requests for the datasources, the Coordinator may again disable segments if they match any configured [drop rules](../operations/rule-configuration.md#drop-rules). Even if segments are enabled by these APIs, you must configure a [load rule](../operations/rule-configuration.md#load-rules) to load them onto Historical processes. If an indexing or kill task runs at the same time these APIs are invoked, the behavior is undefined. Some segments might be killed and others might be enabled. It's also possible that all segments might be disabled, but the indexing task can still read data from those segments and succeed.
```
Not sure if enabled is the same thing as marking a segment as used, and if the DELETE APIs have anything to do with enabling a segment.